pkgs/applications/networking/ids/zeek/default.nix

```diff
@@ -16,28 +19,37 @@
 , gettext
 , coreutils
 , ncurses
-, caf
 }:
-
 stdenv.mkDerivation rec {
   pname = "zeek";
-  version = "4.2.2";
+  version = "5.1.1";
 
   src = fetchurl {
     url = "https://download.zeek.org/zeek-${version}.tar.gz";
-    sha256 = "sha256-9Q3X24uAmnSnLUAklK+gC0Mu8eh81ZE2h/7uIVc8cAw=";
+    sha256 = "sha256-2igxgP3xeNPKYJ0+bb6YggxK7rPib+65ciHPphavxsI=";
   };
 
+  patches = [
+    ./avoid-broken-tests.patch
+    ./debug-runtime-undef-fortify-source.patch
+    ./fix-installation.patch
+  ];
+
   nativeBuildInputs = [
     bison
     cmake
     file
     flex
   ];
 
+  broker = callPackage ./broker { };
+
   buildInputs = [
+    broker
+    spicy-parser-generator
```

Comment:
If the binary is run from it, it must go to nativeBuildInputs.

Reply:
It is used as a JIT at runtime wrapped in spicy-plugin, which implements its own driver.